pybricksdev/connections/pybricks.py

```diff
@@ -447,6 +452,8 @@ async def start_user_program(self) -> None:
             response=True,
         )
 
+        self.monitor_responses()
```

Comment:
There is probably a much better way to do this. Python isn't my primary language -- do you have any ideas, @dlech?